backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -242,6 +246,7 @@ def create_provider_privileges(
         jurisdiction_postal_abbreviations: list[str],
         license_expiration_date: date,
         compact_transaction_id: str,
+        existing_privileges: list[dict],
     ):
         """
```

Comment:
Keyboard Navigation (using VoiceOver): `/aslp/Users`

* When a modal is open, keyboard navigation still goes through links behind the modal. It gets very confusing, and is possible to open up a second modal under the existing modal, e.g.